cmd/broscoli/main_test.go

```diff
@@ -0,0 +1,136 @@
+package main
+
+import (
+	"bufio"
+	"fmt"
+	"github.com/hypebeast/go-osc/osc"
+	"gopkg.in/yaml.v2"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"strings"
+	"testing"
+	"time"
+)
+
+const (
+	SubCmdFlags = "SUB_CMD_FLAGS"
+)
+
+func TestAPI(t *testing.T) {
+	_ = os.Remove("/tmp/foo")
+	file, _ := os.Create("/tmp/bar")
+	defer file.Close()
```

Comment:
You also want to remove the file eventually I reckon?

Reply:
In the end opted in for https://pkg.go.dev/testing#T.TempDir , they promise it'll clean up after itself.